simulation_code/test.jl

```diff
@@ -0,0 +1,72 @@
+
+using Distributions;
+using Distances;
+using Random;
+
+
+function hop_distance(
+        i::Int64,
+        j::Int64,
+        N::Int64
+    )::Int64
+    return min(abs(i-j), abs(j-i), abs(j-i-N), abs(i-j-N))
+end
+
+function cov_mat_fun(sigma::Float64, rho::Float64, N::Int64)::Array{Float64,2}
+    cov_mat = zeros(Float64, N, N)
+    for i in 1:N 
+        for j in 1:N
+            dist = hop_distance( i, j , N)
+            #dist = Distances.euclidean( iota(i,N), iota(j,N) )
+            cov_mat[i,j] = sigma * rho^dist
+        end
+    end
+    return cov_mat
+end
+
+function thompson_sampling(
+    mu_V::Array{Float64,1},
+    Sigma_V::Array{Float64,2},
+    N
+)
+
+
+    draws = [
+            rand(
+                MvNormal(
```

Comment:
this shouldn't be multivariate Normal but rather just a univariate Normal

Reply:
changed to `Normal` in b089f70